lib/index.js

```diff
@@ -36,7 +36,9 @@ TwitterAds.prototype.initialize = function() {
   // load universal website tag
   if (this.options.universalTagPixelId) {
     /* eslint-disable */
-    (function(e,n,u,a){e.twq||(a=e.twq=function(){a.exe?a.exe.apply(a,arguments):a.queue.push(arguments);},a.version='1',a.queue=[])})(window,document,'script');
+    !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe.apply(s,arguments):s.queue.push(arguments);
+},s.version='1.1',s.queue=[],u=t.createElement(n),u.async=!0,u.src='//static.ads-twitter.com/uwt.js',
+a=t.getElementsByTagName(n)[0],a.parentNode.insertBefore(u,a))}(window,document,'script');
```

Comment:
This looks like its going to load the script, which we generally handle ourselves. Could you strip out the script loading part of this?